layers/stateless/parameter_validation_utils.cpp

```diff
@@ -8599,16 +8613,18 @@ bool StatelessValidation::ValidateAccelerationStructureBuildGeometryInfoKHR(
                 if (pInfos[i].type == VK_ACCELERATION_STRUCTURE_TYPE_BOTTOM_LEVEL_KHR) {
                     if (pInfos[i].pGeometries[j].geometryType == VK_GEOMETRY_TYPE_INSTANCES_KHR) {
                         skip |= LogError(device, "VUID-VkAccelerationStructureBuildGeometryInfoKHR-type-03791",
-                                         "%s(): If type is VK_ACCELERATION_STRUCTURE_TYPE_BOTTOM_LEVEL_KHR the geometryType member "
-                                         "of elements of"
-                                         " either pGeometries or ppGeometries must not be VK_GEOMETRY_TYPE_INSTANCES_KHR.",
-                                         api_name);
+                                         "%s(): If pInfos[%" PRIu32
+                                         "].type is VK_ACCELERATION_STRUCTURE_TYPE_BOTTOM_LEVEL_KHR the geometryType member "
+                                         "of elements of pGeometries[%" PRIu32 "] must not be VK_GEOMETRY_TYPE_INSTANCES_KHR.",
+                                         api_name, i, j);
```

Comment:
Nit and probably out of the scope of this PR, but message could be simplified a bit I think.
```suggestion
                                         "%s(): If pInfos[%" PRIu32
                                         "].type is VK_ACCELERATION_STRUCTURE_TYPE_BOTTOM_LEVEL_KHR then  "
                                         "pGeometries[%" PRIu32 "].geometryType must not be VK_GEOMETRY_TYPE_INSTANCES_KHR.",
                                         api_name, i, j);
```

Reply:
I agree, happy for a follow up PR! I just want to get some useful messages in as I hit these errors and was frustrating from the lack of info